README.md

```diff
@@ -1,3 +1,59 @@
 # liftbridge-grpc
 
 Protobuf definitions for the [Liftbridge](https://github.com/liftbridge-io/liftbridge) gRPC API.
+
+## NATS Message Envelope
+
+Every Liftbridge message that is sent over NATS should be sent with the following
+envelope header:
```

Comment:
It's important that we still support "plain" NATS messages for cases where people just want transparent recording of NATS subjects without publisher code changes. The trade-off obviously is that you give up the features of the envelope (key, acking, etc.), but that's the expectation.

Reply:
Yes, absolutely. I should be more clear in the wording here. The plain messages on subjects with streams attached will be treated as value-only messages with no acks etc, just like you have it now. This note is only referring to the Liftbridge-generated messges sent over nats.